bubblewrap.c

```diff
@@ -1096,7 +1103,15 @@ privileged_op (int         privileged_op_socket,
 
     case PRIV_SEP_OP_TMPFS_MOUNT:
       {
-        cleanup_free char *mode = xasprintf ("mode=%#o", perms);
+        cleanup_free char *mode;
+
+        /* TODO: tmpfs(5) seems to interpret a size > 2^64 - 2^12 as "zero"; do
+         * we need to catch that case here? */
```

Comment:
I think it would be safest to limit the argument to a bit less than the range available - 2^63 seems like plenty!